earthmover/operations/column.py

```diff
@@ -82,13 +83,10 @@ def execute(self, data: 'DataFrame', **kwargs) -> 'DataFrame':
         super().execute(data, **kwargs)
 
         for col, val in self.columns_dict.items():
-            # Apply value to all columns
-            if col == "*":
-                for col in data.columns:
-                    self.apply_jinja(data, col, self.columns_dict["*"])
-            # Apply value to specified column
-            else:
-                self.apply_jinja(data, col, val)
+            # Use the fnmatch column functionality to find each valid value
+            for data_column in data.columns:
```

Comment:
Let me know if this approach is on the right track!

The naming convention here may need tweaking. This also allows for sequential matches to also occur ie more than one modification can happen on the same column. Unclear if that is intended behavior.